e2e-tests/pages/dashboardPage.ts

```diff
@@ -186,34 +189,30 @@ export class DashboardPage {
         await this.open()
         await checkAuthState(this.page)
         await this.maybeDeleteMasks()
-        
+
         // create mask and use generated mask email to test email forwarding feature
         await this.generateMask(1)
         const generatedMaskEmail = await this.maskCardGeneratedEmail.textContent()
-            
-        await this.page.goto("https://monitor.firefox.com/")
-    
-        const checkForBreachesEmailInput = this.page.locator('#scan-email').first();
-        const newsLetterCheckBox = '.create-fxa-checkbox-checkmark';
-        const CheckForBreachesButton = this.page.locator('#scan-user-email [data-entrypoint="fx-monitor-check-for-breaches-blue-btn"]').first();
-    
-        await checkForBreachesEmailInput.fill(generatedMaskEmail as string)
-        await this.page.check(newsLetterCheckBox)    
-        await Promise.all([
-          this.page.waitForNavigation(),
-          CheckForBreachesButton.click()
-        ]);
-    
+
+        await this.page.goto("https://monitor.firefox.com/", { waitUntil: 'networkidle' })
```

Comment:
Should we use a different test site instead of Monitor? Something we can control? Something like we did for [Facebook Container](https://maxxcrawford.github.io/social-implementation-examples/facebook/share-dialog/) or about:logins did for the Relay x Firefox integration.

Reply:
Here's the Relay test site: https://mozilla.github.io/form-fill-examples/password_manager/login_and_pw_change_forms.html

Reply:
oooh didn't know about this site, if registering do we get an email or anything of that sort?

Reply:
Yeah, I think that won't work because we need to actually receive the email. Do note that the Monitor site is also undergoing quite a few changes though; hopefully we'll keep a button with a "Get started" label :)

Reply:
I've made updates for the current Monitor design, and a TODO to use a service that we control (maybe we need to create one...)